packages/react-keytips/src/hooks/useTree.ts

```diff
@@ -155,15 +175,19 @@ export function useTree() {
 
   const isCurrentKeytipParent = React.useCallback((keytip: KeytipProps) => {
     if (!currentKeytip.current) return false;
-    const fullSequence = keytip.keySequences.slice(0, -1);
+    const sequence = keytip.keySequences.slice(0, -1) ?? [];
     const parentID =
-      fullSequence.length === 0 ? KTP_ROOT_ID : sequencesToID(fullSequence);
+      sequence.length === 0 ? KTP_ROOT_ID : sequencesToID(sequence);
     return currentKeytip.current.id === parentID;
   }, []);
 
+  const getNode = (id: string) =>
+    [...nodeMap.current.values()].find((node) => node.id === id);
```

Comment:
I know that JS Set doesn't have .find, and for-of might be more verbose, but it will decrease complexity from 2N to N